Examples/Io/Root/include/ActsExamples/Io/Root/RootTrajectoryWriter.hpp

```diff
@@ -126,14 +126,14 @@ class RootTrajectoryWriter final : public WriterT<TrajectoryContainer> {
   std::vector<int> m_dim_hit;       ///< dimension of measurement
 
   bool m_hasFittedParams;        ///< if the track has fitted parameter
-  float m_eLOC0_fit{-99.};       ///< fitted parameter eLOC_0
-  float m_eLOC1_fit{-99.};       ///< fitted parameter eLOC_1
+  float m_eLOC0_fit{-99.};       ///< fitted parameter eBoundLoc0
+  float m_eLOC1_fit{-99.};       ///< fitted parameter eBoundLoc1
   float m_ePHI_fit{-99.};        ///< fitted parameter ePHI
   float m_eTHETA_fit{-99.};      ///< fitted parameter eTHETA
   float m_eQOP_fit{-99.};        ///< fitted parameter eQOP
   float m_eT_fit{-99.};          ///< fitted parameter eT
   float m_err_eLOC0_fit{-99.};   ///< fitted parameter eLOC_-99.err
```

Comment:
Ha, could you help fix the comment here?

Reply:
Done.